### ISSUE TITLE
Circular dependencies in growth/discovery

### ISSUE THREAD

**Issue by tomlinton** [OriginProtocol/origin#2454]
Output from `lerna` below. Although there are three errors it seems like probably only one issue.

```
lerna WARN ECYCLE Dependency cycles detected, you should fix these!
lerna WARN ECYCLE @origin/cron -> @origin/growth -> @origin/discovery -> @origin/growth
lerna WARN ECYCLE @origin/discovery -> @origin/growth -> @origin/discovery
lerna WARN ECYCLE @origin/growth -> @origin/discovery -> @origin/growth
```

**Comment by shahthepro** [comment]
@tomlinton Moving Growth Events to a separate package removed the `@origin/discovery`'s dependency on `@origin/growth`. So this is fixed as a side effect of PR #2759

**Comment by tomlinton** [comment]
Awesome, thanks @shahthepro :)
